src/DoctrineExtensions.php

```diff
@@ -36,15 +37,19 @@ final class DoctrineExtensions
      */
     public static function registerMappingIntoDriverChainORM(MappingDriverChain $driverChain, ?Reader $reader = null): void
     {
-        if (!$reader) {
-            $reader = self::createAnnotationReader();
-        }
-        $annotationDriver = new DriverORM\AnnotationDriver($reader, [
+        $paths = [
             __DIR__.'/Translatable/Entity',
             __DIR__.'/Loggable/Entity',
             __DIR__.'/Tree/Entity',
-        ]);
-        $driverChain->addDriver($annotationDriver, 'Gedmo');
+        ];
+
+        if (PHP_VERSION_ID >= 80000) {
```

Comment:
I suggest using `\PHP_VERSION_ID` or `use const PHP_VERSION_ID` (but I'm not sure the second version is supported in all PHP versions supported by this library) instead of relying on the fallback to the global namespace. For a qualified constant coming from the core, PHP will resolve it at compile time, which will then allow OPCache to resolve the whole `if` at compile time.

Reply:
Done.